### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Finniki**
# Description

This pr adds HTML and CSS codes.

# Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Details

- ### **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

- ### **What is the current behavior?** (You can also link to an open issue here)
The user can read the content

## Screenshots (if applicable)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  <body>
+      <main>
+        <div class="right-div">
+          <address class="address   address-right">
+            <strong class="strong-text">Dr. Eleanor Gaye</strong><br/>
+            Awesome Science faculty<br/>
+            University of Awesome<br/>
+            Bobtown, CA 99999,<br/>
+            USA<br/>
+            <strong   class="strong-text">Tel:</strong> 123-456-7890<br/>
```
Comment (gertie-sheshe) [inline index.html]:
```suggestion
            <strong class="strong-text">Tel:</strong> 123-456-7890<br/>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400&display=swap" rel="stylesheet">
+  <title>Marking Up A Letter</title>
+  </head>
+  <body>
+      <main>
+        <div class="right-div">
+          <address class="address   address-right">
```
Comment (gertie-sheshe) [inline index.html]:
```suggestion
          <address class="address address-right">
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        <p>Dear Eileen,</p>
+
+        <p>Thank you for your recent application to join us at the University of Awesome's science faculty to study as part of your <span class="text-underline">PhD</span> next year. I will answer your questions one by one, in the following sections.</p>
+
+        <section>
+          <h2 class="section-header">Starting dates</h2>
+
+          <p>We are happy to accommodate you starting your study with us at any time, however it would suit us better if you could start at the beginning of a semester; the start dates for each one are as follows:</p>
+          <ul>
+            <li             class="list-item">First semester: <time datetime="2016-09-09"> 9 September 2016</time>
```
Comment (gertie-sheshe) [inline index.html]:
```suggestion
            <li class="list-item">First semester: <time datetime="2016-09-09"> 9 September 2016</time>
```

**Review by gertie-sheshe — COMMENTED** [review]
Great work 🎉 @Finniki 

What IDE are you using? You seem to be missing a code formatter

File: index.html (hunk 3 above)
Comment (Finniki) [reply index.html]: Totally missed those… Thanks

**Pull request merged**